inference-engine/src/mkldnn_plugin/mkldnn_graph.cpp

```diff
@@ -1228,3 +1238,8 @@ void MKLDNNGraph::printGraphInfo() const {
         std::cout << " ]"  << std::endl;
     }
 }
+
+void MKLDNNGraph::checkFuncQuantized(std::shared_ptr<const ngraph::Function> func) {
```

Comment:
The method responsibilities are not the same as stated in the name. Moreover it allows to call the method for ngraph function not associated with this particular CPU graph. We need to change the semantics or remove it at all.

Reply:
Since the method is two lines long, I decided to inline it.